nipoppy/workflows/runner.py

```diff
@@ -208,7 +210,17 @@ def run_single(self, participant_id: str, session_id: str):
 
     def run_cleanup(self):
         """Run pipeline runner cleanup."""
-        for dpath in [self.dpath_pipeline_bids_db, self.dpath_pipeline_work]:
-            if dpath.exists():
-                self.rm(dpath)
+        if self.n_success == self.n_total:
+            self.logger.info("All pipeline segments completed successfully.")
```

Comment:
Don't think this is needed since there is already a feedback message printed (from [here](https://github.com/nipoppy/nipoppy/blob/bc0f04ae5d7878c105d0228443d192acfd8b18a6/nipoppy/workflows/pipeline.py#L393-L398))